tailwind-showcases/src/TailwindShowcases.tsx

```diff
@@ -44,46 +43,86 @@ const getClassSuffixes = (theme: Record<string, any>) => ({
 });
 
 const getClassNames = (suffixes: Record<string, string[]>) => ({
-  backgroundColor: { component: 'box', classes: suffixes.colors.map(s => `bg${s}`) },
-  opacity: { component: 'box', classes: suffixes.opacity.map(s => `opacity${s}`) },
-  shadow: { component: 'box', classes: suffixes.boxShadow.map(s => `shadow${s}`) },
-  borderRadius: { component: 'box', classes: suffixes.borderRadius.map(s => `rounded${s}`) },
-  borderWidth: { component: 'box', classes: suffixes.borderWidth.map(s => `border${  s}`) },
-  borderColor: { component: 'box', classes: suffixes.colors.map(s => `border${s}`) },
-  fontFamily: { component: 'text', classes: suffixes.fontFamily.map(s => `font${s}`) },
-  fontSize: { component: 'text', classes: suffixes.fontSize.map(s => `text${s}`) },
-  fontWeight: { component: 'text', classes: suffixes.fontWeight.map(s => `font${s}`) },
-  textColor: { component: 'text', classes: suffixes.colors.map(s => `text${s}`) },
-  letterSpacing: { component: 'text', classes: suffixes.letterSpacing.map(s => `tracking${s}`) },
-  lineHeight: { component: 'text', classes: suffixes.lineHeight.map(s => `leading${s}`) },
+  backgroundColor: {
+    component: 'box',
+    classes: suffixes.colors.map((s) => `bg${s}`),
```

Comment:
for later we could maybe destructure inside the param to increase typing/readabilty ?